### ISSUE TITLE
Very poor repaint on linux window resize

### ISSUE THREAD

**Issue by llamadonica** [domokit/mojo#237]
There's a lot of flickering on linux window resizes. It eventually seems to redraw correctly, but looks bad while doing it.

**Comment by abarth** [comment]
Yeah, the linux build isn't meant to be particularly polished.  The window represents the entire screen on a real device.  Resizing the window is akin to switching screens, which is why it's a very heavy-weight operation.  I'm not sure how much we're going to invest in polishing the desktop linux configuration.

@jamesr
